frontend/src/report/report-page/report-header/ReportHeader.svelte

```diff
@@ -42,8 +42,8 @@
 			variant="outlined"
 			bind:value={currentReport.name}
 			label="Report Name"
-			input$maxlength={15}>
-			<CharacterCounter slot="helper">0 / 15</CharacterCounter>
+			input$maxlength={30}>
+			<CharacterCounter slot="helper">0 / 30</CharacterCounter>
```

Comment:
Do we need a max length at all? I guess this is good for rendering to make sure it looks nice in the titles and stuff. I'm okay with it I think

Reply:
hmm actually I've hidden the overflow, so we could remove the max length, the format should stay the same no matter how many characters the users have input
![image](https://user-images.githubusercontent.com/89871868/226407876-01a010df-7edb-4e19-9b0c-fa7813c96e2c.png)